ui/app/components/app/connected-sites-list/connected-sites-list.component.js

```diff
@@ -23,25 +26,31 @@ export default class ConnectedSitesList extends Component {
     return (
       <main className="connected-sites-list__content-rows">
         { connectedDomains.map((domain) => (
-          <div key={domain.key} className="connected-sites-list__content-row">
+          <div key={domain.origin} className="connected-sites-list__content-row">
             <div className="connected-sites-list__domain-info">
               <IconWithFallBack icon={domain.icon} name={domain.name} />
-              <span className="connected-sites-list__domain-name" title={domain.extensionId || domain.key}>
-                {
-                  domain.extensionId
-                    ? t('externalExtension')
-                    : domain.key
-                }
+              <span className="connected-sites-list__domain-name" title={domain.extensionId || domain.origin}>
+                {this.getDomainDisplayName(domain)}
               </span>
             </div>
             <i
               className="fas fa-trash-alt connected-sites-list__trash"
               title={t('disconnect')}
-              onClick={() => onDisconnect(domain.key)}
+              onClick={() => onDisconnect(domain.origin)}
             />
           </div>
         )) }
       </main>
     )
   }
+
+  getDomainDisplayName (domain) {
+    if (domain.extensionId) {
+      return domain.extensionId
```

Comment:
It looks like you've changed the name used for extensions:
```suggestion
      return this.context.t('externalExtension')
```

Reply:
Tsk, thought about making sure to do that then promptly forgot about it.